beacon-chain.k

```diff
@@ -334,7 +377,7 @@ def compute_activation_exit_epoch(epoch: Epoch) -> Epoch:
     return Epoch(epoch + 1 + ACTIVATION_EXIT_DELAY)
 */
   syntax Int ::= "compute_activation_exit_epoch" "(" Int ")" [function]
-  rule compute_activation_exit_epoch(EP) => EP +Int 1 +Int ACTIVATION_EXIT_DELAY
```

Comment:
Can you also update python code in the comment, here and in other places?

Reply:
I updated the comment here. I couldn't find any other places I missed though, do you still see any?

Reply:
Thanks! I didn't see any other places missing. Just wrote a general message. Will review soon.